PacketDotNet/IpPacket.cs

```diff
@@ -121,16 +121,6 @@ public override Packet PayloadPacket
         /// </value>
         public abstract IPProtocolType Protocol { get; set; }
 
-        /// <value>
```

Comment:
Please keep to not break existing code.

Reply:
I'll added it back, but override it in IPv4 with an Obsolete tag. It's not an IPv4 concept.